.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 - git reset --hard
 
 script:
+- set -e
 - make generate
 - if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run
   `make generate`. Please run it and commit the changes"; git status --porcelain; false; fi
```

Comment:
Nice , i would suggest to add some special reserved word like
TRAVIS_ERROR on each of the error cases,
So in case we will back to "run all" in the future, it will be still easier to track the error,
Running all cost resources, hard to find (unless there is a special word),
but takes less iteration to see all the errors at once, if we run just one by one, it might take more iterations,
to see all the error, and sometimes you do want to know all at once, to get the whole picture imo.

Reply:
Next time please comment on the reasoning in the first PR comment.

If this change will be accepted, the future PR which will want to add "run all" will need to explain and resolve the problems it introduces.
I want to solve some of the problems this current "run all" causes.

Reply:
np, if i understand you right, i marked this line
so we can add TRAVIS_ERROR on each error, but up to you.